plugins/org.jkiss.dbeaver.ui.editors.sql/src/org/jkiss/dbeaver/ui/editors/sql/handlers/SQLEditorHandlerOpenObjectConsole.java

```diff
@@ -109,15 +109,35 @@ void openConsole(IWorkbenchWindow workbenchWindow, DBRRunnableWithResult<String>
 
         openAndExecuteSQLScript(workbenchWindow, navigatorContext, title, doRun, currentSelection, sql);
     }
-
+    
     public static void openAndExecuteSQLScript(
+            IWorkbenchWindow workbenchWindow,
+            SQLNavigatorContext navigatorContext,
+            String title,
+            boolean doRun,
+            ISelection currentSelection,
+            String sql) throws CoreException {
+        openAndExecuteSQLScriptExt(
+            workbenchWindow,
+            navigatorContext,
+            title,
+            doRun,
+            currentSelection,
+            sql, 
+            false);
+    }
+
+    /**
+     * openAndExecuteSQLScript with force sqlScript executing
+     */
+    public static void openAndExecuteSQLScriptExt(
         IWorkbenchWindow workbenchWindow,
         SQLNavigatorContext navigatorContext,
         String title,
         boolean doRun,
         ISelection currentSelection,
-        String sql) throws CoreException
-    {
+        String sql,
+        boolean forceScript) throws CoreException {
```

Comment:
Clarify the parameter name and fix line wraps

```suggestion
        boolean forceProcessAsScript
    ) throws CoreException {
```